extensions/fop/runtime/src/main/doc/limitations.adoc

```diff
@@ -0,0 +1,12 @@
+While you can use any of the available output types in JVM mode, only PDF output type is supported
+in native mode. PDF output type in native mode has several limitations:
+
+* Default sRGB color space is disabled because of https://github.com/oracle/graal/issues/2850[Graal VM issue #2850]. This limitation makes configuration property
+`disable-srgb-colorspace` ignored. You can see more about sRGB color space in
+https://xmlgraphics.apache.org/fop/2.1/configuration.html[FOP configuration].
```

Comment:
```suggestion
* Default sRGB color space is always disabled because of https://github.com/oracle/graal/issues/2850[Graal VM issue #2850] and `disable-srgb-colorspace` https://xmlgraphics.apache.org/fop/2.1/configuration.html[FOP configuration property] is ignored.
```

Reply:
done